### PR TITLE
[chore][scehams] - make it a go pkg

### PR DESCRIPTION
**Pull request by VihasMakwana**
This will be consumed by `meshery/server` once released

**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by VihasMakwana** [comment]
@leecalcote @theBeginner86 is this what you guys are expecting? Let me know.

File: go.mod
Comment (leecalcote) [inline go.mod:1]: Can we make this meshery/schemas?

**Review by MUzairS15 — APPROVED** [review]
Will you also open a PR in meshkit to remove the copy of this code?

**Review by Yashsharma1911 — COMMENTED** [review]
@VihasMakwana sounds good, however you might not need to import these schemas we already have them here inside `/schemas` directory here is one of example https://github.com/meshery/schemas/blob/master/schemas/external/ui/catalog/applicationImport.json

A proper structure to store schemas is needed, you might like to check schemas directory in the root directory, the folder structure of schemas in Meshkit isn't good

**Review by MUzairS15 — COMMENTED** [review]
As a reminder, this repository is intended to store schemas and their equivalent golang structs, and not any kind of core logic around it. Hence I prefer to keep the logic to refer these RJSF schemas (required by UI) in `meshkit`, i.e. update the [function](https://github.com/meshery/meshkit/blob/0878150d8145d0cace27e600c44dc66adda961b0/schemas/schemaProvider.go#L32) to refer the schemas defined in this `schemas` repo using a git walker.

**Comment by VihasMakwana** [comment]
> Will you also open a PR in meshkit to remove the copy of this code? https://github.com/meshery/meshkit/blob/0878150d8145d0cace27e600c44dc66adda961b0/schemas/schemaProvider.go#L32

Yup

**Review by theBeginner86 — APPROVED** [review]

**Comment by leecalcote** [comment]
There has been another commit or two made. Ready for a second review?

**Comment by VihasMakwana** [comment]
@leecalcote @MUzairS15 yeah, PTAL. Thansks

**Review by MUzairS15 — COMMENTED** [review]
Thank you for volunteering @VihasMakwana , we have decided to move forward with approach. The RJSF schemas that were being added as a package will now be served form the sistent NPM package.

**Comment by leecalcote** [comment]
Given this I'll propose that we close this PR. At the same time, I'll say: @VihasMakwana, thanks for this PR! Come back. We miss you. 😉

**Pull request closed without merging**